backend-external/src/v1/routes/pay-transparency-routes.ts

```diff
@@ -142,15 +142,15 @@ const validateApiKey =
  *       - in: query
  *         name: startDate
  *         type: date
- *         pattern: /([0-9]{4})-(?:[0-9]{2})-([0-9]{2})/
+ *         pattern: /([0-9]{4})-(?:[0-9]{2})-([0-9]{2})\s([0-9]{2}):([0-9]{2}):([0-9]{2})/
  *         required: false
- *         description: "Start date in UTC for the update date range filter (format: YYYY-MM-dd, default -31 days ) - optional"
+ *         description: "Start date in UTC for the update date range filter (format: YYYY-MM-dd HH:mm:ss, default -31 days ) - optional"
```

Comment:
The requirement was to input HH:mm, not HH:mm:ss. They didn't want to use seconds. Would it work to append ':00' to valid the input?